src/Controller/PagesController.php

```diff
@@ -39,10 +40,12 @@ public function index(): Response
         $manager = $this->getDoctrine();
         $postRepository = $manager->getRepository(Post::class);
         $observationRepository = $manager->getRepository(Observation::class);
+        $featuredSpeciesService = new FeaturedSpecies($manager->getManager());
```

Comment:
Inject service Symfony way : `function index(FeaturedSpecies $featuredSpecies)`

Reply:
will be done in next PR